lib/structures/Invite.js

```diff
@@ -48,6 +49,20 @@ class Invite extends Base {
         this._createdAt = data.created_at !== undefined ? data.created_at : null;
         this.presenceCount = data.approximate_presence_count !== undefined ? data.approximate_presence_count : null;
         this.memberCount = data.approximate_member_count !== undefined ? data.approximate_member_count : null;
+        if(data.stage_instance !== undefined) {
+            data.stage_instance.members = data.stage_instance.members.map((m) => {
+                m.id = m.user.id;
+                return m;
```

Comment:
```suggestion
                return this.guild.members.update(m, this.guild);
```

Reply:
That shouldn't matter though should it? Since Discord sends the guild structure if there is an associated stage instance (unless I'm mistaken)

Reply:
Yeah but said guild structure doesn't include a `shard` property, so it results in an error. This is just what I've been experiencing tbh.

Reply:
Ah I see what you mean now

Reply:
Yeah, that's why I tried to make it so it doesn't try and update the members if the guild isn't cached, but maybe there's a better way of doing it

Reply:
Should be fixed